[2025-04-12T21:00:00.910+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-12T21:00:00.953+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_gera_csv.gera_csv manual__2025-04-12T20:59:57.403587+00:00 [queued]>
[2025-04-12T21:00:00.968+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_gera_csv.gera_csv manual__2025-04-12T20:59:57.403587+00:00 [queued]>
[2025-04-12T21:00:00.969+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-12T21:00:00.996+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): gera_csv> on 2025-04-12 20:59:57.403587+00:00
[2025-04-12T21:00:01.008+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=76) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-12T21:00:01.011+0000] {standard_task_runner.py:72} INFO - Started process 78 to run task
[2025-04-12T21:00:01.011+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_gera_csv', 'gera_csv', 'manual__2025-04-12T20:59:57.403587+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpscas6txn']
[2025-04-12T21:00:01.016+0000] {standard_task_runner.py:105} INFO - Job 31: Subtask gera_csv
[2025-04-12T21:00:01.365+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_gera_csv.gera_csv manual__2025-04-12T20:59:57.403587+00:00 [running]> on host d1d0c8246bed
[2025-04-12T21:00:01.535+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='MÃ¡rio Kojima' AIRFLOW_CTX_DAG_ID='dag_gera_csv' AIRFLOW_CTX_TASK_ID='gera_csv' AIRFLOW_CTX_EXECUTION_DATE='2025-04-12T20:59:57.403587+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-12T20:59:57.403587+00:00'
[2025-04-12T21:00:01.537+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-12T21:00:01.597+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-12T21:00:01.622+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-12T21:00:01.623+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_gera_csv, task_id=gera_csv, run_id=manual__2025-04-12T20:59:57.403587+00:00, execution_date=20250412T205957, start_date=20250412T210000, end_date=20250412T210001
[2025-04-12T21:00:01.752+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-12T21:00:01.792+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-12T21:00:01.795+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
